### ISSUE TITLE
isEditable of "SCIHorizontalLineAnnotation" is not working in SciChart v3.x

### ISSUE THREAD

**Issue by bernardIos** [ABTSoftware/SciChart.iOS.Examples#9]
I have used the SciChart v3.x library into the project and I have added the SCIHorizontalLineAnnotation to the surface but isEditable is not working in v3.x. I have checked  this functionality into the v2.x and working fine but had issue in v3.x. see the lines that I have done. isEditable is not working even though in your demo as well.

```
// SciChart v3.x
        let horizontalLine = SCIHorizontalLineAnnotation()
        horizontalLine.set(x1: 0.0)
        horizontalLine.set(y1: 0.2)
        horizontalLine.isEditable = true
        horizontalLine.horizontalAlignment = .right
        horizontalLine.stroke = SCISolidPenStyle(color: UIColor.orange, thickness: 2)

       let lineAnnotationLabel = SCIAnnotationLabel()
        lineAnnotationLabel.text = "Trigger"
        lineAnnotationLabel.labelPlacement = .topLeft
        horizontalLine.annotationLabels.add(lineAnnotationLabel)

       SCIUpdateSuspender.usingWith(surface) {
            self.surface.xAxes.add(self.xAxis)
            self.surface.yAxes.add(self.yAxis)
            self.surface.renderableSeries.add(self.rSeries)
            self.surface.chartModifiers.add(modifierGroup)
            self.surface.annotations.add(horizontalLine)
        }
```

**Comment by andriypohorilko** [comment]
Hi, there. 
Thank you for your request. 
We've checked your code and can't reproduce this issue. The "isEditable" functionality works. You can drag and resize an annotation. Please see a video:
https://www.dropbox.com/s/gxiuu2mx07kxaxt/Annotation_editing.mov?dl=0

In case you mean some other functionality that you were using in SciChart v2, please describe it in more detail and provide some code snippets, screenshots, etc.

Thanks in advance.

**Comment by bernardIos** [comment]
@andriypohorilko I have tested the V3.x Example app's annotations are easy and interaction with annotations in both drag and resize an annotations are not working. I am using a chartModifiers as well into the app so may be any conflict. can you provide me a your demo code snippets?.

**Comment by andyb1979** [comment]
Hi @bernardIos and @andriypohorilko 

It might be a conflict between gesture handling of some modifiers. I'm testing now the SciChart iOS v3.x Examples app (straight from the app store). the example 'Interaction With Annotations' allows dragging and editing of annotations, and has a ZoomPanModifier but does not have PinchZoomModifier

**Comment by andyb1979** [comment]
@bernardIos can you share some code that highlights the issue, or make one of our examples break by adding functionality? If so share it here, we will investigate.

**Comment by bernardIos** [comment]
Hi @andyb1979 and @andriypohorilko
Thanks for your response. I have installed the app from App Store and it works  on iPhone 13.4.1. However, when I install the app from this link:  https://github.com/ABTSoftware/SciChart.iOS.Examples
And run the app on Xcode 11.3.1  into my device, anotations stops working. I am not able to drag and drop. Please see this video:
https://drive.google.com/file/d/1J0CLnz4TaQLhjfppl6VZxw1YWp1KLS8Q/view?usp=sharing
As you can see first one is from app stroe and it works fine but the second one is from github (above link) is not edditable. In the video might not be very much visible but i’m tapping on all the corners and trying to select and drag and drop but it is not working! Please let me know if you need any more information.
Thanks,
Bernard

**Comment by andriypohorilko** [comment]
Hi, Bernard.
Unfortunately, I still can't reproduce this issue on the example app. Probably, something is wrong with the SciChart cocoapods installation.
Please, try this project: https://www.dropbox.com/s/a8gm7bmifvk35vk/Examples.zip?dl=0
Cocoapods are already installed there. You just need to enter your license key and run the project.
It works on my iPhone 8, 13.4.1 

Alternatively, try this:
1) Download fresh zip from https://github.com/ABTSoftware/SciChart.iOS.Examples
2) run "pod repo update" and "pod install" in project folder terminal

Let us know the result.
Thanks in advance

**Comment by andyb1979** [comment]
@bernardIos if the above doesnt work can you let us know iPhone model and ios version please

**Comment by bernardIos** [comment]
Hi @andyb1979 and @andriypohorilko 

Thanks for response and sending the files. 

Unfortunately it did not work for me. I have downloaded source code from dropbox only added my licence number and changed the apple sign-in certificate and run the app on my phone. I did not run "Pod install" as it already had pod files.

Also I tied alternative way as explained and installed the pods. Same issue as demonstrated in the video occurred again.

I'm using:

Xcode 11.3.1 
iPhone XS Max and iPhone 8
iOS 13.4.1

Just to clarify when I download the [app](https://apps.apple.com/au/app/scichart-examples/id1485200499) from AppStore, it worked fine with no issue.
